cdf_fabric_replicator/extractor.py

```diff
@@ -1,17 +1,22 @@
 import time
```

Comment:
Im not sure i understand why we have extractor.py in the cdf_fabric_replicator folder if a cdf_fabric_extractor folder already exists.

Reply:
That was their mistake they corrected want everything in the replicator directory now